### ISSUE TITLE
'Punctual receiver intensity' file/results wit SPPS, is made of zeros

### ISSUE THREAD

**Issue by Picaut** [Ifsttar/I-Simpa#256]
**Describe the bug**
The table obtained with SPPS for the Punctual receiver intensity is made of zeros

**To Reproduce**
1. Create a valide project (a room, a sound source, a punctual receiver, and material)
2. Run SPPS
3. Goto to results and open 'Punctual Receiver intensity' in the corresponding folder

**Expected behavior**
We should have valid values in the table (may be a problem of number of decimals?)

**Screenshots**
See screenshot

**Desktop (please complete the following information):**
 - OS: Windows
 - Version I-SIMPA 1.3.4, Built on 2020-06-26T08:52:28

**Additional context**
When running the 'Intensity animation', we can see valid vector.

![image](https://user-images.githubusercontent.com/7996615/101057778-b5663500-358c-11eb-8bc7-3760806f53ee.png)

**Comment by nicolas-f** [comment]
Yes it's number of decimal. There is no "scientific notation" for numbers. Some power should be set to sound sources in order to see something here. The original file does not have this issue as numbers are stored in binary.
